packages/@uppy/companion/test/__tests__/uploader.js

```diff
@@ -190,7 +190,7 @@ describe('uploader with tus protocol', () => {
   })
 
   // eslint-disable-next-line max-len
-  const formDataNoMetaMatch = /^----------------------------\d+\r\nContent-Disposition: form-data; name="files\[\]"; filename="uppy-file-[^"]+"\r\nContent-Type: application\/octet-stream\r\n\r\nSome file content\r\n----------------------------\d+--\r\n$/
+  const formDataNoMetaMatch = /^--form-data-boundary-[a-z0-9]+\r\nContent-Disposition: form-data; name="files\[\]"; filename="uppy-file-[^"]+"\r\nContent-Type: application\/octet-stream\r\n\r\n$/
```

Comment:
The fact that file content in no longer in the body is a bit worrying, not sure why that happens 🤔

Reply:
Definitely related to the Nock patch, thankfully we can disable the patch for these test cases